seed/views/00601_team_court_reporter_overview.js

```diff
@@ -0,0 +1,24 @@
+exports.seed = function (knex, promise) {
+  var sql = `CREATE VIEW app.team_court_reporter_overview
+    WITH SCHEMABINDING
+    AS
+    SELECT
+        MAX(CONCAT(om.forename, ' ', om.surname)) AS name
+      , MAX(iv.grade_code) AS grade_code
+      , MAX(iv.contracted_hours) AS contracted_hours
+      , MAX(iv.reduction_hours) AS reduction_hours
+      , MAX(iv.link_id) AS id
+      , MAX(iv.id) AS link_id
+      , SUM(iv.total_cases_sdrs) AS total_cases_sdrs
+      , SUM(iv.total_cases_fdrs) AS total_cases_fdrs
+      , SUM(iv.total_cases_oral_reports) AS total_cases_oral_reports
+    FROM app.individual_court_reporter_overview iv
```

Comment:
These views will need to be refactored to not reference each other to work with the new indexes